### DISCUSSION TITLE
Item Listing Request for SKU with whitespace / space

### DISCUSSION THREAD

**Discussion by loneregister** [amzn/selling-partner-api-models#3301] [General]
What is the proper URL and URI for the /listings/2021-08-01/items/{sellerId}/{sku} API call for a sku that has a white space in it.

For example a SKU of "THIS IS A SKU"?

Currenly, I have a functioning API that works just fine for all SKUs without spaces.  But fails with a "THIS%20IS%20A%20SKU" does not exist in marketplace.... when in fact it does.  Albeit, with " ", and not "%20".

This is all via php (fyi).
Thank you

**Comment by loneregister** [comment]
It also appears that a / will cause issues as well.
From my perspective it appears that anything that will break a URL, will break this API.
I suggest that the SKU be changed to the querystring to prevent this issue.

**Comment by chapmanjw** [comment]
@LoneRegister Are you still experiencing this issue? We have rolled out a few encoding-related fixes and SKUs with spaces are now working within our own canary tests. If you are still experiencing this issue, please contact Developer Support with a request id and timestamp so that we can troubleshoot further on our end. Thanks!
